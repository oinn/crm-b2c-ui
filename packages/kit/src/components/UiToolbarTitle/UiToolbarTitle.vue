<template>
  <div class="ui-toolbar-title">
    <slot></slot>
  </div>
</template>

<script lang="ts">
import { defineComponent } from 'vue';

export default defineComponent({
  name: 'UiToolbarTitle',
});
</script>

<style lang="scss">
.ui-toolbar-title {
  font-size: 16px;
  font-style: normal;
  font-weight: 700;
  line-height: 22px;
}
</style>
