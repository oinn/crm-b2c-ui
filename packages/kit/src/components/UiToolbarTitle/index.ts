export { default as UiToolbarTitle } from './UiToolbarTitle.vue';
