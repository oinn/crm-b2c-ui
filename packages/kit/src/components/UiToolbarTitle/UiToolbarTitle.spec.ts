import { mount } from '@vue/test-utils';
import UiToolbarTitle from './UiToolbarTitle.vue';

describe('UiToolbarTitle', () => {
  it('Rendered correctly', () => {
    const wrapper = mount(UiToolbarTitle, {
      slots: {
        default: 'Default slot text',
      },
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
