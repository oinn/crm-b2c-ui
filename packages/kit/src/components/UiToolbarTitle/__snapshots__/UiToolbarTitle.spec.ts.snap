// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UiToolbarTitle Rendered correctly 1`] = `"<div class=\\"ui-toolbar-title\\">Default slot text</div>"`;
